src/test/scala/edu/ie3/simona/event/listener/ResultEventListenerSpec.scala

```diff
@@ -141,6 +137,22 @@ class ResultEventListenerSpec
         assert(outputFile.exists)
         assert(outputFile.isFile)
       }
+
+      "check if actor dies when it should die" in {
+        val fileHierarchy = resultFileHierarchy(1, ".ttt")
```

Comment:
```suggestion
        val fileHierarchy = resultFileHierarchy(5, ".ttt")
```

Reply:
Since the specific runid is used for a `ResultFileHierarchy` at line 121 the test can fail since the directory that should be created already exists. As the direcetory that is created uses the runid passed to the `resultFileHierarchy` for naming using a different id solves the problem

Reply:
Already fixed this in the branch I mentioned [here](https://github.com/ie3-institute/simona/pull/171#issuecomment-1065615202)